src/blk/pmem/PMEMDevice.cc

```diff
@@ -54,10 +59,61 @@ int PMEMDevice::_lock()
   return 0;
 }
 
+static int pmem_check_file_type(int fd, const char *pmem_file, uint64_t *total_size)
+{
+  int rc = 0;
+  struct stat file_stat;
+  char spath[PATH_MAX];
+  char npath[PATH_MAX];
+  char *real_path, *base_name;
```

Comment:
please define variable close to where it is used for the first time for better readability.

Reply:
ack